pkg/unikontainers/unikontainers.go

```diff
@@ -427,18 +433,23 @@ func (u *Unikontainer) saveContainerState() error {
 	return os.WriteFile(stateName, data, 0o644) //nolint: gosec
 }
 
-// ExecuteHooks executes concurrently any hooks found in spec based on name:
 func (u *Unikontainer) ExecuteHooks(name string) error {
+	return u.executeHooksConcurrently(name)
+	// return u.executeHooksSequentially(name)
+}
+
+// ExecuteHooks executes concurrently any hooks found in spec based on name:
+func (u *Unikontainer) executeHooksConcurrently(name string) error {
 	// NOTICE: It is possible that the concurrent execution of the hooks may cause
 	// some unknown problems down the line. Be sure to prioritize checking with sequential
 	// hook execution when debugging.
 
 	// More info for individual hooks can be found here:
 	// https://github.com/opencontainers/runtime-spec/blob/main/config.md#posix-platform-hooks
+	Log.WithField("hooks", name).Error("")
```

Comment:
Let's remove this, or add it at debug level, with a small message.

Reply:
You are right, I have changed the log level.